resources/install/debian/postinst

```diff
@@ -96,8 +96,23 @@ case "$1" in
         # and this breaks startup script
         sed -i 's/\$JVB_EXTRA_JVM_PARAMS//g' $CONFIG
 
-        if ! grep -q "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
-            echo "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.${JVB_HOSTNAME}/.*" >> $NEW_JITSI_CONFIG
+        if ! grep -q "#org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
```

Comment:
Is the `if` necessary? The only case where it makes a difference is when the file contains both the commented-out and non-commented-out version.

Does this script execute when the debian package is updated?

Reply:
Yes, this is executed on every update. The goal of this is to make current configs work(migrate them) and give people an option to anyway have something in there and we do not override it on update(having commented and not commented line). It is not documented but it is an option if someone complains about something like that.

Reply:
Okay..